massa-execution-exports/src/event_store.rs

```diff
@@ -234,7 +234,7 @@ impl EventStore {
             .iter()
             // filter on slots
             .filter_map(|(slot, ids)| {
-                if slot >= &start && slot < &end {
+                if slot >= &start && slot <= &end {
```

Comment:
Why did you changed here ?

Reply:
It looks like the documentation didn't have information about how that slots are used. Should I fix that?

Reply:
Yeah if you can. :)

Reply:
done

Reply:
done